spec/models/notification_type_spec.rb

```diff
@@ -1,46 +1,57 @@
 describe NotificationType, :type => :model do
   describe '.seed' do
```

Comment:
We have shared examples for this, see `include_examples(".seed called multiple times", 20)`

Reply:
If I'm reading that right I would have to hardcode the number of seeded types into the spec ... right? I don't really want to do that.

Reply:
Yes, any reason not to?

Reply:
I was thinking because this changes much more frequently than the other classes that use that shared example.  After a quick look the fixture has a commit about once a month for the last year or so.

Reply:
I'll change it if you feel strongly about it though. I was also just leaning towards not changing the specs that don't apply to what I was actually fixing (outside of making this file comply with the rspec conventions the rest of the project uses).

Reply:
It's fine, you left it better than it was :smile: